src/main/java/pro/belbix/ethparser/repositories/HardWorkRepository.java

```diff
@@ -61,7 +61,9 @@
         + "select sum(saved_gas_fees) from hard_work where vault = :vault and block_date < :blockDate")
     Double sumSavedGasFees(@Param("vault") String vault, @Param("blockDate") long blockDate);
 
-    List<HardWorkDTO> findAllByVaultOrderByBlockDate(String vault);
+    @Query("select t from HardWorkDTO t where t.vault = :vault and t.blockDate between :startTime and :endTime order by t.blockDate")
```

Comment:
I like HQL `between` usage!
it will return an inclusive range, some of my requests are not inclusive
we have protection from not unique data on the UI side so it is not a problem
let's use `between` for any further features